src/components/Class/index.js

```diff
@@ -353,21 +366,31 @@ function Class({ classToEdit, indicator }) {
                       setFormField("doubt_class", TYPE);
                     }}
                     value={formFieldsState[TYPE]}
-                    id="type1"
+                    id="type2"
                     checked={
                       formFieldsState.type === "doubt_class" ? "checked" : false
                     }
+                    disabled={
+                      isEditMode
+                        ? formFieldsState[TYPE] === "doubt_class"
+                          ? false
+                          : true
+                        : false
+                    }
```

Comment:
Similar comment here:
```suggestion
                    disabled={isEditMode && formFieldsState[TYPE] !== "doubt_class"}
```
But also lines 336-359 and lines 360-383 are almost identical so you may want to consider creating a new Component to avoid this duplication.